security/infisical/Portfile

```
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        infisical infisical 0.28.4 infisical-cli/v
+github.tarball_from releases
+revision            0
+
+categories          security devel
+maintainers         openmaintainer {esafak @esafak}
+license             MIT
+
+description         Official CLI for Infisical, an open source secrets platform
+long_description    Official CLI for Infisical, an open source secrets platform
```

Nitpick, but perhaps next time:

```suggestion
long_description    ${description}
```

---

Done